src/widget/wtracktableview.cpp

```diff
@@ -565,6 +570,14 @@ void WTrackTableView::mouseMoveEvent(QMouseEvent* pEvent) {
     if (!trackModel) {
         return;
     }
+
+    const auto delta = m_mousePressPosition - pEvent->position();
```

Comment:
The type of auto is out of sight here. Can you make it explicit?

Reply:
done